annex/destroy.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 docker stop montagu_db_annex
+docker rm montagu_db_annex
```

Comment:
My guess for the addition of the `rm` commands - we _used_ to have `docker run --rm` but we've since made it a service (with `--restart=always`) and that's incompatible with `--rm` so we removed that but never added cleanup

Reply:
here are relevant commits: https://github.com/vimc/montagu-db/commit/1bd1ec1fb51ca9f92a52b1cae7a1ea6c5947b01 (daemonising) https://github.com/vimc/montagu-db/commit/13c00c0ed92524aff087fd7f220fc7caf745d353 (removing `--rm`)